### PR TITLE
Standardization

### PR DESCRIPTION
**Pull request by StuntsPT**
Documentation standardizations, better "Standard JS" conformance, bumped package versions and implemented "Standard JS" tests.

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -6,150 +6,133 @@
...
-
-                for (var i = 0; i < numSamples; i++) {
-                    sampleIndex[i] = sampleinfo[9 + i];
-                }
-
-            }
-
-
+    if (line.indexOf('#') === 0) {
+            // ##fileformat=VCFv4.1
```
Comment (bmpvieira) [inline lib/index.js]: Can you please fix the indentation of the comments here and below?

File: package.json [hunk 2]
```diff
@@ -22,16 +22,18 @@
   ],
   "main": "lib/index.js",
   "scripts": {
-    "test": "mocha"
+    "test": "standard && dependency-check ."
```
Comment (bmpvieira) [inline package.json]: You should keep the mocha testing here, like `standard && dependency-check . && mocha`

**Review by bmpvieira — CHANGES_REQUESTED** [review]
Looks good, just needs two small changes.

**Comment by StuntsPT** [comment]
These two commits should handle your concerns. Unless I got the indentation wrong...

**Review by bmpvieira — APPROVED** [review]

**Pull request merged**